src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, T value);
```

Comment:
The `fill` method should use the generic type parameter `T` instead of `Machine` for the `machines` list. For example: `void fill(List<? super T> machines, T value);`